slides/01-introduction.qmd

```diff
@@ -324,9 +340,9 @@ abalone |>
 #| dev-args: list(bg = "transparent")
 tree_mod <- 
   rpart::rpart(
-    rings ~ .,
-    data = abalone,
-    control = rpart::rpart.control(maxdepth = 3, cp = 0.001)
+    results ~ facility_type + risk + total_violations,
+    data = inspection,
+    #control = rpart::rpart.control(maxdepth = 3, cp = 0.001)
```

Comment:
not sure if the control is still relevant...this plot is not as useful with this dataset

Reply:
I got this to give us a nice-ish plot (nice enough to talk over about training a model) 👍